perfkitbenchmarker/benchmarks/iperf_benchmark.py

```diff
@@ -24,10 +24,13 @@
 import re
 
 from perfkitbenchmarker import flags
-from perfkitbenchmarker import regex_util
 from perfkitbenchmarker import sample
 from perfkitbenchmarker import vm_util
 
+flags.DEFINE_integer('sending_thread_count', 1,
```

Comment:
I thought about this in terms of naming. We should have done this from the
beginning. let me fix it.
On May 9, 2015 10:00 AM, "Connor McCoy" <notifications@github.com> wrote:

> In perfkitbenchmarker/benchmarks/iperf_benchmark.py
> <https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/pull/262#discussion_r29994360>
> :
>
> >  from perfkitbenchmarker import sample
> >  from perfkitbenchmarker import vm_util
> >
> > +flags.DEFINE_integer('sending_thread_count', 1,
>
> Maybe iperf_sending_thread_count?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/pull/262/files#r29994360>
> .
>